.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @snyk/lumos
+* @snyk/unify
```

Comment:
@ChristinaDara One more thing to note as part of passing over the code ownership is to add `unify` team to the repo with an `Admin` role.

Reply:
ye s I did give them an admin and removed the old teams from there